### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Minderheiten**

### CODE REVIEW

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 1
- Score: 0%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 0
- Score: 0%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 1
- Score: 0%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 1
- Score: 0%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 1
- Score: 0%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 105
- Score: 97.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 106
- Score: 98.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 107
- Score: 99.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 107
- Score: 99.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 107
- Score: 99.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 107
- Score: 99.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Review by iphysresearch — APPROVED** [review]
@Minderheiten 
恭喜完成Python编程基础作业，要不把扩展作业的5个题也做一下吧哈？

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Minderheiten
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by Minderheiten** [comment]
leetcode扩展作业
[LeetCode_homework.md](https://github.com/iphysresearch/GWData-Bootcamp/files/13784068/LeetCode_homework.md)

**Comment by Minderheiten** [comment]
leetcode编程扩展作业 :D
[LeetCode_homework.md](https://github.com/iphysresearch/GWData-Bootcamp/files/13784072/LeetCode_homework.md)
@iphysresearch

File: 2023/homework/HanPeijia/homework_seaborn.ipynb
Comment (iphysresearch) [inline 2023/homework/HanPeijia/homework_seaborn.ipynb:1]: 完成seaborn可视化作业！

**Pull request merged**